sdcm/sct_config.py

```diff
@@ -688,7 +689,7 @@ class SCTConfiguration(dict):
         dict(name="security_group_ids", env="SCT_SECURITY_GROUP_IDS", type=str_or_list,
              help="AWS security groups ids to use"),
 
-        dict(name="use_placement_group", env="SCT_USE_PLACEMENT_GROUP", type=str,
+        dict(name="use_placement_group", env="SCT_USE_PLACEMENT_GROUP", type=boolean,
```

Comment:
This issue is also interesting, I wonder how it works we didn't see placement groups being used everywhere?

Reply:
since we always set it from yaml files, and not from env variables